source/configuration.txt

```diff
@@ -1,16 +1,160 @@
+
+To learn more about .NET project files, see 
+`Understanding the Project File <https://docs.microsoft.com/en-us/aspnet/web-forms/overview/deployment/web-deployment-in-the-enterprise/understanding-the-project-file>`__
+from Microsoft.
+
+To learn more about the ``AdditionalFiles`` tag, see 
+:github:`Using Additional Files </dotnet/roslyn/blob/main/docs/analyzers/Using%20Additional%20Files.md>`
+in the Roslyn documentation.
+
+Options
+-------
+
+The following is a description of all configuration options supported
+by the {+product+}:
+
+.. list-table::
+   :header-rows: 1
+   :widths: 35 65
+
+   * - Name
+     - Description
+
+   * - | **DefaultLinqVersion**
+     - | **Type:** string
+       |
+       | **Description:**
+       | The LINQ provider the {+product+} uses.
+         
+       .. tip::
+
+          To learn more about LINQ, see the
+          :ref:`Analyze Your Code <analyzer-analyze-linq>`
+          page.
+
+       | **Default**: ``"V2"``
+       | **Accepted Values**: ``"V2"`` or ``"V3"``
+
+   * - | **OutputDriverVersion**
+     - | **Type:** boolean
+       |
+       | **Description:**
+       | Specifies if the {+product+} includes your {+driver-short+} version in diagnostic messages.
+       |
+       | **Default**: ``false``
+       | **Accepted Values**: ``true`` or ``false``
+
+   * - | **OutputInternalExceptions**
+     - | **Type:** boolean
+       |
+       | **Description:**
+       | Specifies if the {+product+} includes internal exceptions in diagnostic messages.
+       |
+       | **Default**: ``false``
+       | **Accepted Values**: ``true`` or ``false``
+
+   * - | **OutputInternalLogsToFile**
+     - | **Type:** boolean
+       |
+       | **Description:**
+       | Specifies if the {+product+} writes its internal logs to a file.
+
+       .. include:: includes/logging-admonition.rst       
+
+       | **Default**: ``false``
+       | **Accepted Values**: ``true`` or ``false``
+
+   * - | **LogFileName**
+     - | **Type:** string
+       |
+       | **Description:**
+       | Specifies the path to which the {+product+} writes its internal logs.
+       
+       .. include:: includes/logging-admonition.rst       
+
+       | **Default**: ``""``
+       | **Accepted Values**: A valid file path
+   * - | **SendTelemetry**
+     - | **Type:** boolean
+       |
+       | **Description:**
+       | Specifies if the {+product+} sends anonymized information to MongoDB (the company).
+         To learn more, see the :ref:`telemetry <mongodb-analyzer-configuration-telemetry>`
+         section of this guide.
+
+       | **Default**: ``true``
+       | **Accepted Values**: ``true`` or ``false``
+
+
+To learn more about the configuration options the {+product+} supports,
+see the :github:`{+product+} source code </master/src/MongoDB.Analyzer/Core/Settings/SettingsHelper.cs>`.
+
+.. _mongodb-analyzer-configuration-telemetry:
+
+Telemetry
+~~~~~~~~~
+
+When the {+product+} analyzes your source code, the tool by default
```

Comment:
question: same here about "(the company)" - would stating the why and removing MongoDB avoid this? I think most users just need to know that data is being sent and what it's for.

Reply:
I think "who is my data sent to?" is an important question to answer as well as "why is my data sent?". As stated in preceding response, I don't have a great citation for why this is important. If people on the team feel strongly that this question should not be answered here then I agree that "MongoDB Inc./MongoDB (the company)" does not have to be mentioned.

Reply:
My main point was that MongoDB (the company) was breaking from convention.